Common/BaselineSelection.py

```diff
@@ -133,149 +133,89 @@ def DefineMETCuts(met_thr, met_collections):
+            && (Tau_idDeepTau{deepTauVersions[config["deepTauVersion"]]}v{config["deepTauVersion"]}VSmu >= {getattr(WorkingPointsTauVSmu, config["deepTauWPs"][ch]["VSmu"]).value})'''
+        df = df.Define(f'Tau_B2_{ch}_2', cut_str)
+        if ch == 'tauTau':
+            cut_str += f' && (Tau_idDeepTau{deepTauVersions[config["deepTauVersion"]]}v{config["deepTauVersion"]}VSjet >= {getattr(WorkingPointsTauVSjet, config["deepTauWPs"]["tauTau"]["VSjet"]).value})'
+            df = df.Define(f'Tau_B2_{ch}_1', cut_str)
 
-    df = df.Define("Tau_B2_tauTau_2", f"""
-        Tau_B0 && v_ops::pt(Tau_p4) > 20 && v_ops::eta(Tau_p4) < {eta_cut}
-        && (Tau_idDeepTau{deepTauVersions[config["deepTauVersion"]]}v{config["deepTauVersion"]}VSe >= {getattr(WorkingPointsTauVSe, config["deepTauWPs"]["tauTau"]["VSe"]).value})
-        && (Tau_idDeepTau{deepTauVersions[config["deepTauVersion"]]}v{config["deepTauVersion"]}VSmu >= {getattr(WorkingPointsTauVSmu, config["deepTauWPs"]["tauTau"]["VSmu"]).value})
-    """)
 
     df = df.Define("Muon_B2_muMu_1", f"""
-        Muon_B0 && v_ops::pt(Muon_p4) > 15 && (   (Muon_tightId && Muon_pfRelIso04_all < 0.15)
-                                    || (Muon_highPtId && Muon_tkRelIso < 0.15) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.15) || (Muon_highPtId && Muon_tkRelIso < 0.15) )
     """)
     df = df.Define("Muon_B2_muMu_2", f"""
-        Muon_B0 && v_ops::pt(Muon_p4) > 15 && (   (Muon_tightId && Muon_pfRelIso04_all < 0.3)
-                                    || (Muon_highPtId && Muon_tkRelIso < 0.3) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.3) || (Muon_highPtId && Muon_tkRelIso < 0.3) )
     """)
 
-    df = df.Define("Electron_B2_eMu_1", f"""
-        Electron_B0 && v_ops::pt(Electron_p4) > 10 && Electron_mvaNoIso_WP80 && Electron_pfRelIso03_all < 0.3
-    """)
+    df = df.Define("Electron_B2_eMu_1",f"Electron_B0 && Electron_mvaIso_WP80 ")
+    #  Electron_mvaNoIso_WP80 && Electron_pfRelIso03_all < 0.3
     df = df.Define("Muon_B2_eMu_2", f"""
-        Muon_B0 && v_ops::pt(Muon_p4) > 15 && (   (Muon_tightId && Muon_pfRelIso04_all < 0.15)
-                                    || (Muon_highPtId && Muon_tkRelIso < 0.15) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.3) || (Muon_highPtId && Muon_tkRelIso < 0.3) )
     """)
 
-    df = df.Define("Electron_B2_eE_1", f"""
-        Electron_B0 && v_ops::pt(Electron_p4) > 10 &&  Electron_mvaNoIso_WP80 && Electron_pfRelIso03_all < 0.15
-    """)
-    df = df.Define("Electron_B2_eE_2", f"""
-                    Electron_B2_eE_1
-    """)
+    df = df.Define("Electron_B2_eE_1",f"Electron_B0 && Electron_mvaIso_WP80 ")
+    #  Electron_mvaNoIso_WP80 && Electron_pfRelIso03_all < 0.3
+    df = df.Define("Electron_B2_eE_2", f""" Electron_B2_eE_1 """)
```

Comment:
We can't apply `mvaIso` for both legs in `eE`. I suggest applying `mvaNoIso` and `pfRelIso < 0.3` for the second leg.